src/services/authService.js

```
@@ -0,0 +1,179 @@
+    const token = this.getToken();
+    if (!token) {
+      return false;
+    }
+
+    // 토큰 만료 시간 확인 (JWT 토큰인 경우)
+    try {
+      if (token.includes('.')) {
+        const payload = JSON.parse(atob(token.split('.')[1]));
+        const currentTime = Date.now() / 1000;
+        
+        if (payload.exp && payload.exp < currentTime) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)